lib/openers/pdf-view-opener.js

```diff
@@ -3,24 +3,34 @@
 import Opener from '../opener'
 import { isPdfFile } from '../werkzeug'
 
-function forwardSync (pdfView, texPath, lineNumber) {
-  if (pdfView != null && pdfView.forwardSync != null) {
-    pdfView.forwardSync(texPath, lineNumber)
-  }
+function sleep (time) {
+  return new Promise(resolve => setTimeout(resolve, time))
 }
 
 export default class PdfViewOpener extends Opener {
   async open (filePath, texPath, lineNumber) {
-    const openPaneItems = atom.workspace.getPaneItems()
-    for (const openPaneItem of openPaneItems) {
-      if (openPaneItem.filePath === filePath) {
-        forwardSync(openPaneItem, texPath, lineNumber)
-        return true
-      }
+    const texPane = atom.workspace.paneForURI(texPath)
+
+    // This prevents splitting the right pane multiple times
+    if (texPane) {
+      texPane.activate()
     }
 
-    // TODO: Make this configurable?
-    atom.workspace.open(filePath, {'split': 'right'}).then(pane => forwardSync(pane, texPath, lineNumber))
+    const options = {
+      searchAllPanes: true,
+      split: atom.config.get('latex.openerSplit')
+    }
+
+    const item = await atom.workspace.open(filePath, options)
+    const pane = atom.workspace.paneForItem(item)
+    if (pane && item && item.forwardSync) {
+      item.forwardSync(texPath, lineNumber)
+      // If we don't wait for pdf-view to do it's first update then it will just
+      // display a blank page when there are multiple jobs.
+      while (item.updating) {
+        await sleep(100)
```

Comment:
I agree. It's just a temporary hack. I'll submit a PR to `pdf-view` and leave this PR open until there is an obvious resolution. They were pretty fast with the last PR I submitted. It will probably have to be done with a callback or `EventEmitter` though. There are some time delays used in `pdf-view`.